test-framework/core/src/main/java/org/keycloak/testframework/https/ManagedCertificates.java

```diff
@@ -0,0 +1,117 @@
+package org.keycloak.testframework.https;
+
+import org.jboss.logging.Logger;
+import org.keycloak.common.crypto.CryptoIntegration;
+import org.keycloak.common.crypto.CryptoProvider;
+import org.keycloak.common.util.KeystoreUtil;
+import org.keycloak.crypto.def.DefaultCryptoProvider;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.KeyPair;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+public class ManagedCertificates {
+
+    private static final Logger LOGGER = Logger.getLogger(ManagedCertificates.class);
+
+    private final CryptoProvider cryptoProvider;
+
+    private KeyStore keyStore;
+    private X509Certificate certificate;
+    private KeyPair keyPair;
+
+    private final static Path KEYSTORE_FILE_PATH = Paths.get(System.getProperty("java.io.tmpdir"), "kc-server-testing.keystore");
+    public final static char[] KEYSTORE_PASSWORD = "password".toCharArray();
+    public final static char[] PRVKEY_PASSWORD = "password".toCharArray();
+
+    public final static String CERT_ENTRY = "cert";
+    public final static String PRV_KEY_ENTRY = "prvKey";
+
+
+    public ManagedCertificates() throws ManagedCertificatesException {
+        cryptoProvider = new DefaultCryptoProvider();
+        if (!CryptoIntegration.isInitialised()) {
+            CryptoIntegration.setProvider(cryptoProvider);
+        }
+        initKeyStore();
+    }
+
+    public Path getKeycloakServerKeyStorePath() {
+        return KEYSTORE_FILE_PATH;
+    }
+
+    public KeyStore getKeyStore() {
+        return keyStore;
+    }
```

Comment:
This should not exist as we don't need access to the KeyStore directly, what we need is `public KeyStore getClientTrustStore()`.